packages/cli/src/Server.ts

```diff
@@ -268,11 +266,24 @@ class App {
 		this.presetCredentialsLoaded = false;
 		this.endpointPresetCredentials = config.get('credentials.overwrite.endpoint') as string;
 
-		// TODO UM: remove this flag
-		this.isUserManagementEnabled = !config.get('userManagement.disabled');
+		this.frontendSettings = this.getCurrentFrontendSettings();
+	}
 
-		const urlBaseWebhook = WebhookHelpers.getWebhookBaseUrl();
+	/**
+	 * Returns the current epoch time
+	 *
+	 * @returns {number}
+	 * @memberof App
+	 */
+	getCurrentDate(): Date {
+		return new Date();
+	}
 
+	/**
+	 * Returns the current settings for the frontend
+	 */
+	getCurrentFrontendSettings(): IN8nUISettings {
```

Comment:
Nitpick - `getCurrentFrontendSettings` sounds like the settings are _from_ the FE, instead of being settings _for_ the FE. And "current" seems superfluous, when are they not current? In short, what about `getSettingsForFrontend`?